### ISSUE TITLE
MVP Proposal Comments

### ISSUE THREAD

**Issue by loschorts** [email2jie/Bellenista#1]
Hey Jie,

Great proposal! Here are a few comments:
- Budget more time for Front-End Auth. It can be tricky.
- I am a little confused about `Product` vs `ProductDetail`. Can you clarify? Is `Product` the Index.
- Your routes seem a bit confusing. If I were you I'd do it like this.

```
    <Route path="/" component={App}/>
        <IndexRoute component={ProductCategory}>
        <Route path="category/:category" component={ProductCategory}>
        <Route path="product/:productID" component={ProductDetail}  

```

Then render a `ProductIndex` full of `ProductIndexItems` in `ProductCategory`.

**Comment by email2jie** [comment]
Yes I was thinking Product to be more like Index and ProductDetail to be ProductIndexItems. But I wasn't sure whether to call it a product index or not because products are displayed by their category. So we are rendering only products that has a category rather than all index of a product.
